packages/contracts-bedrock/src/L2/SuperchainTokenBridge.sol

```diff
@@ -63,6 +67,8 @@ contract SuperchainTokenBridge {
     {
```

Comment:
You could imagine in the future, this also could support other types of tokens, which may require a new entrypoint into this contract or it could overload this functionality

Reply:
Deferring to discussion here on Discord: https://discord.com/channels/1244729134312198194/1303074442490023936/1303093468117078078